pkg/apiserver/common/common.go

```diff
@@ -65,7 +65,7 @@ func Register(r *gin.RouterGroup, s *Service) {
 	endpoint := r.Group("/common")
 
 	endpoint.POST("/pods", s.listPods)
-	endpoint.GET("/namespaces", s.getNamespaces)
+	endpoint.GET("/chaos-available-namespaces", s.getChaosAvailableNamespaces)
```

Comment:
how about keep the `/namespaces` and add the new API `/chaos-available-namespaces`

Reply:
Great! I am working on this.

Reply:
Hi @WangXiangUSTC, it finished in 4a056c7 , please take a look.